include/ginkgo/core/base/array.hpp

```diff
@@ -354,6 +380,8 @@ class Array {
 
     /**
      * Resizes the array so it is able to hold the specified number of elements.
+     * To a view, this only resets the pointer. Afterwards, the  only way to use
+     * a view in such a state is to `std::move` another view.
```

Comment:
You could also move another array into it. I would write something like:
`After a reset of a view, the view can be moved to in order to use it afterwards.`